gulpfile.babel.js

```diff
@@ -290,6 +292,18 @@ function lintBootstrap() {
 		}));
 }
 
+export function lintMarkup() {
+	if (isProdBuild()) {
+		return gulp.src([...settings.sources.markup, './*.html'])
+		.pipe(htmllint())
+		.pipe(gulp.dest(settings.destinations.dev.markup));
```

Comment:
I wonder that this task has a `dest`.
I know it doesn’t matter. But I think it should be `settings.destinations.prod.markup` for the production build.